### PR TITLE
Set TELEPORT_ETCD_TEST=yes.

### PR DESCRIPTION
**Pull request by Tener**
As far as I can tell, we have a bug in our current code:

```go
	cmd.Env = append(os.Environ(), "TELEPORT_ETCD_TEST=yes")
	cmd.Env = append(os.Environ(), "TELEPORT_XAUTH_TEST=yes")
```
Given the above, `cmd.Env` will **not** have "TELEPORT_ETCD_TEST=yes" set.

### CODE REVIEW

**Review by zmb3 — APPROVED** [review]
Good catch!

File: .cloudbuild/scripts/cmd/unit-tests/main.go [hunk 1]
```diff
@@ -179,8 +179,9 @@ func run() error {
 func runUnitTests(workspace string) error {
 	cmd := exec.Command("make", "test")
 	cmd.Dir = workspace
-	cmd.Env = append(os.Environ(), "TELEPORT_ETCD_TEST=yes")
-	cmd.Env = append(os.Environ(), "TELEPORT_XAUTH_TEST=yes")
+	cmd.Env = os.Environ()
+	cmd.Env = append(cmd.Env, "TELEPORT_ETCD_TEST=yes")
+	cmd.Env = append(cmd.Env, "TELEPORT_XAUTH_TEST=yes")
```
Comment (codingllama) [inline .cloudbuild/scripts/cmd/unit-tests/main.go:184]:
```suggestion
	cmd.Env = append(os.Environ(), "TELEPORT_ETCD_TEST=yes", "TELEPORT_XAUTH_TEST=yes")
```

less lines = less bugs, literally in this case. :)

**Review by codingllama — APPROVED** [review]
Well spotted, thanks.

File: .cloudbuild/scripts/cmd/unit-tests/main.go (hunk 1 above)
Comment (Tener) [reply .cloudbuild/scripts/cmd/unit-tests/main.go:184]: I was thinking about that, but at least in a few places saw the appends split into individual lines. I think that is more `git-blame` friendly? Not sure if there are other rationales for one or the other style.

**Review by marcoandredinis — APPROVED** [review]

**Pull request merged**